api/admin/controller/sign_in.py

```diff
@@ -94,7 +96,9 @@ def ekirjasto_auth_finish(self):
         if isinstance(user_info, ProblemDetail):
             return user_info
 
-        circulation_role = self._to_circulation_role(user_info.role)
+        circulation_role = self._to_circulation_role(
+            self._db, user_info.role, user_info.municipality
```

Comment:
This part is still pending some discussion/decisions on how the two municipality fields (`municipality` and `municipalities`) we get during authentication (with user info) will be managed.

Reply:
This is now handled. The singular `municipality` field will be used for patrons and `municipalities` field for admins and librarians to determine which libraries they can manage.